### ISSUE TITLE
In the Subsonic settings the Password field is misleading when using MD5 token

### ISSUE THREAD

**Issue by cquike** [strawberrymusicplayer/strawberry#874]
**Describe the bug**

In the authentication field of the subsonic settings there are two fields: Username and Password. However the second one (Password) is actually misleading if one chooses MD5 token as authentication method (which an be selected below that).

 I think the authentication method should be displayed before and if one selects MD5 token the Password field should be named API key, otherwise it is confusing for the user.

**System Information:**
 - Operating system: Linux
 - Strawberry Version:

**Comment by jonaski** [comment]
I don't understand what you mean with this. Subsonic supports two authentication methods, both uses the username and password. There is no API key used. A MD5 is created from the password using a random salt.

**Comment by cquike** [comment]
I see. I think I got confused by the Ampache API key (https://github.com/ampache/ampache/wiki/subsonic).
Thanks!

**Comment by cquike** [comment]
After more investigation, apart from  my confusion, and after some more testing I did actually found a bug. As of a bit of context: ampache has subsonic support but the password to use is different whether you are using MD5 token configuration or not. So I had my account in Strawberry properly setup with Hex method. Then I tried to test the MD5 method and after changing the password I clicked on Test and it never worked.
The bug is actually that when you click on test it uses the previous password that was saved, ignoring any change you have now made to the password.
